### DISCUSSION TITLE
What is the package manager of mojo?

### DISCUSSION THREAD

**Discussion by tanganke** [modular/modular#1171] [Q&A] [answered]
Is there a package manager or package repository such as pip and pypi?

**Comment by david-ragazzi** [comment]
> [Modular CLI tool](https://docs.modular.com/cli/), which works like a package manager to install and update Mojo.

https://docs.modular.com/cli/

**Comment by Sharktheone** [comment, marked as answer]
Currently there is `modular` which is mainly used for installing mojo, but probably this or the `mojo` command it self is going to be the package manager for mojo in the future. But you can also use pip for the "normal" python packages with the python interface.

**Comment by hhstore** [comment]
- hi, all.
- I have written a package management tool called `Mod(= Mojo Dep)` for Mojo (WIP).
- The `mod new` command has been implemented so far.
- Other commands are still under development.
- Feel free to join in the development if you're interested. 😄
- 🔥 https://github.com/better-mojo/mod
- 🔥 https://pypi.org/project/mod-cli/
